packages/__docs__/src/Params/index.tsx

```diff
@@ -55,7 +55,7 @@ class Params extends Component<ParamsProps> {
   }
 
   renderType(type?: { names: string[] }) {
-    return type ? type.names : null
+    return type ? type.names.join(' | ') : null
```

Comment:
this fixes method params written as one large string, e.g. https://instructure.design/#findFocusable